cfgov/scripts/export_enforcement_actions.py

```diff
@@ -38,33 +30,31 @@ def assemble_output():
             continue
         page_categories = ','.join(
             c.get_name_display() for c in page.categories.all())
+        content = ''
+        soup = BeautifulSoup(str(page.content), 'html.parser')
+        para = soup.findAll(['p', 'h5'])
+        for p in para:
+            content += p.get_text()
+            link = p.find('a', href=True)
```

Comment:
This code assumes that there's no more than one anchor tag per `p`. That's probably a safe assumption, but it might be helpful to put a comment here in case someone is troubleshooting later. Something like `<p> tags won't have multiple links, per style guide`.